### PR TITLE
Tenable - Chunk size change and log level change

### PR DESCRIPTION
**Pull request by v-jayakal**
-----------------------------------------------------------------------------------------------------------
## Before submitting this PR please ensure that you have read the following sections and filled out the changes, reason for change and testing complete sections:

Thank you for your contribution to the Microsoft Sentinel Github repo.

> Details of the code changes in your submitted PR.  Providing descriptions for pull requests ensures there is context to changes being made and greatly enhances the code review process.  Providing associated Issues that this resolves also easily connects the reason.
   
   Change(s):
   - Updated syntax for XYZ.yaml

   Reason for Change(s):
   - New schema used for XYZ.yaml
   - Resolves ISSUE #1234

> The code should have been tested in a Microsoft Sentinel environment that does not have any custom parsers, functions or tables, so that you validate no incorrect syntax and execution functions properly.  If your submission requires a custom parser or function, it must be submitted with the PR.

   Testing Completed:
   - Yes/No/Need Help

_Note: If updating a detection, you must update the version field._

> After the submission has been made, please look at the Validation Checks.

   Checked that the validations are passing and have addressed any issues that are present:
   - Yes/No/Need Help
   
   _Note: Let us know if you have tried fixing the validation error and need help._

> References: 
> - [Guidance for Detection checks](https://github.com/Azure/Azure-Sentinel#pull-request-detection-template-structure-validation-check)
> - [General contribution guidance](https://github.com/Azure/Azure-Sentinel/wiki#what-can-you-contribute-and-how-can-you-create-contributions)
> - [PR validation troubleshooting](https://github.com/Azure/Azure-Sentinel#pull-request)

-----------------------------------------------------------------------------------------------------------

### CODE REVIEW

File: Solutions/TenableIO/Data Connectors/host.json [hunk 1]
```diff
@@ -1,6 +1,10 @@
 {
   "version": "2.0",
   "logging": {
+	"logLevel": {
```
Comment (tomeara) [inline Solutions/TenableIO/Data Connectors/host.json:4]: Spacing looks like it should be fixed here

File: Solutions/TenableIO/Data Connectors/TenableStartVulnExportJob/__init__.py [hunk 2]
```diff
@@ -8,7 +8,7 @@ def main(timestamp: int) -> object:
     tio = TenableIO()
     logging.info(
         f'requesting a new Vuln Export Job from Tenable')
-    job_id = tio.exports.vulns(updated_at=timestamp)
+    job_id = tio.exports.vulns(updated_at=timestamp, num_assets=100)
```
Comment (tomeara) [inline Solutions/TenableIO/Data Connectors/TenableStartVulnExportJob/__init__.py:11]: Can we have `num_assets` default to 100, but be configurable to allow adjustment if necessary?

**Review by tomeara — CHANGES_REQUESTED** [review]
Looks like a small spacing issue needs to be fixed, and I raised a question regarding `num_asset` configuration.

File: Solutions/TenableIO/Data Connectors/host.json (hunk 1 above)
Comment (v-jayakal) [reply Solutions/TenableIO/Data Connectors/host.json]: Fixed, package (zip file) does not have this spacing issue.

File: Solutions/TenableIO/Data Connectors/TenableStartVulnExportJob/__init__.py (hunk 2 above)
Comment (v-jayakal) [reply Solutions/TenableIO/Data Connectors/TenableStartVulnExportJob/__init__.py:11]: @tomeara - Can we do that once the issue is resolved?  Even when we reduced the count to 100, customer is still getting the same error. Have you reached to customer using the customer support request they raised? Do you have any findings? Please let us know. Thanks.

**Comment by v-sabiraj** [comment]
One of the ICM is dependent on this PR, once the ICM is resolved we can close/merge this PR.

**Comment by NikTripathi** [comment]
Closing this one as new PR with latest changes has been raised. https://github.com/Azure/Azure-Sentinel/pull/4654

**Pull request closed without merging**